image/stub/stub.go

```diff
@@ -25,6 +25,11 @@ type Stub struct {
 	t image.Type
 }
 
+// Next returns the next extent.
+func (img *Stub) Next(start, length int64) (image.Extent, error) {
```

Comment:
Should this validate `start` and `length`?

Reply:
Yes, like the qcow2 version.

Reply:
Actually not needed since this always fail now. We cannot report extents when the size of the image is unknown and the image is not readable.